projects/kit/components/text-area/text-area.style.less

```diff
@@ -135,6 +135,10 @@
     :host._has-value._has-tooltip & {
         margin-right: 2.5rem;
     }
+
+    &__extra {
+        background-color: var(--tui-error-bg-night);
```

Comment:
Why night? Shouldn't it be --tui-error-bg by default and night for tuiMode onDark?

Reply:
In Figma templates it is `--tui-error-bg-night` for default theme.
<img width="1495" alt="error-bg-night" src="https://user-images.githubusercontent.com/35179038/127989988-7c4359c3-5b30-43bb-a7a4-74222a211a8d.png">

Reply:
Ok, let's keep it that way